src/fsutils.ts

```diff
@@ -0,0 +1,17 @@
+import * as fs from "fs";
```

Comment:
How about

    import {statSync} from "fs";

Reply:
that is possible... I go back and forth haha